src/xai_grok/schemas/tokenize_text_response.py

```diff
@@ -0,0 +1,7 @@
+import typing
+
+import pydantic
+
+
+class TokenizeTextResponse(pydantic.BaseModel):
+    token_ids: typing.List[str]
```

Comment:
The `token_ids` field is defined as a list of strings, but token IDs are typically integers. This will cause type errors when processing actual token IDs from tokenizers.